logstash-core/src/main/java/org/logstash/ext/JRubyWrappedWriteClientExt.java

```diff
@@ -70,27 +69,43 @@ public JRubyWrappedWriteClientExt initialize(final ThreadContext context,
     }
 
     public JRubyWrappedWriteClientExt initialize(
-        final JRubyAbstractQueueWriteClientExt queueWriteClientExt, final String pipelineId,
-        final AbstractMetricExt metric, final IRubyObject pluginId) {
+        final JRubyAbstractQueueWriteClientExt queueWriteClientExt,
+        final String pipelineId,
+        final AbstractMetricExt metric,
+        final IRubyObject pluginId) {
         this.writeClient = queueWriteClientExt;
         // Synchronize on the metric since setting up new fields on it is not threadsafe
         synchronized (metric) {
             final AbstractNamespacedMetricExt eventsMetrics =
-                getMetric(metric, "stats", "events");
+                getMetric(metric,
+                        MetricKeys.STATS_KEY.asJavaString(),
+                        MetricKeys.EVENTS_KEY.asJavaString());
+
             eventsMetricsCounter = LongCounter.fromRubyBase(eventsMetrics, MetricKeys.IN_KEY);
-            eventsMetricsTime = LongCounter.fromRubyBase(eventsMetrics, PUSH_DURATION_KEY);
-            final AbstractNamespacedMetricExt pipelineMetrics =
-                getMetric(metric, "stats", "pipelines", pipelineId, "events");
-            pipelineMetricsCounter = LongCounter.fromRubyBase(pipelineMetrics, MetricKeys.IN_KEY);
-            pipelineMetricsTime = LongCounter.fromRubyBase(pipelineMetrics, PUSH_DURATION_KEY);
+            eventsMetricsTime = LongCounter.fromRubyBase(eventsMetrics, MetricKeys.PUSH_DURATION_KEY);
+
+            final AbstractNamespacedMetricExt pipelineEventMetrics =
+                getMetric(metric, MetricKeys.STATS_KEY.asJavaString(),
+                        MetricKeys.PIPELINES_KEY.asJavaString(),
+                        pipelineId,
+                        MetricKeys.EVENTS_KEY.asJavaString());
```

Comment:
consistency nitpick: I prefer args to be either all-on-one-line or for each to have its own line aligned with the first argument, because I find arbitrary line-breaks difficult to read.

```suggestion
                getMetric(metric,
                          MetricKeys.STATS_KEY.asJavaString(),
                          MetricKeys.PIPELINES_KEY.asJavaString(),
                          pipelineId,
                          MetricKeys.EVENTS_KEY.asJavaString());
```